webpack.config.js

```diff
@@ -27,7 +27,7 @@ const config = {
                     MiniCssExtractPlugin.loader,
                     {
                         loader: 'css-loader',
-                        options: { sourceMap: true, minimize: true }
+                        options: { sourceMap: true }
```

Comment:
@aminroosta note that `minimize` will be set to `true` when webpack is in production mode